packages/core/utils/src/modules-sdk/loaders/load-models.ts

```diff
@@ -29,7 +29,7 @@ export function loadModels(basePath: string) {
         try {
           const required = require(filePath)
           return Object.values(required).filter(
-            (resource) => typeof resource === "function" && !!resource.name
+            (resource: any) => !!resource.name
```

Comment:
Same here, can we maybe type it closer to what it represents?

Reply:
We choose any because a user can decide to export anything he wants and we can't know, so any seamed the most appropriate. I ve changed it still